packages/manager/src/features/CloudPulse/Alerts/AlertsDetail/AlertDetailChips.tsx

```diff
@@ -0,0 +1,99 @@
+import { Typography } from '@linode/ui';
+import { Grid, useTheme } from '@mui/material';
+import React from 'react';
+
+import { StyledAlertChip } from './AlertDetail';
+
+export interface AlertDimensionsProp {
+  /*
+   * The chips that needs to be displayed
+   */
+  chips: Array<string> | Array<string[]>;
+  /*
+   * Controls whether we need to display the chips one by one or join and display
+   */
+  isJoin?: boolean;
+  /*
+   * The label under which the chips will be displayed
+   */
+  label: string;
+  /*
+   * Controls the size of the chip label from medium to larger screens
+   */
+  mdLabel?: number;
+  /*
+   * Controls the size of the chip value from medium to larger screens
+   */
+  mdValue?: number;
```

Comment:
```suggestion
  labelWidth?: number;
  /*
   * Controls the size of the chip value from medium to larger screens
   */
  valueWidth?: number;
```

can we rename it to more understandable name